src/plugins/irc-events/link.js

```diff
@@ -64,7 +64,7 @@ function parse(msg, url, res, client) {
 	case "image/gif":
 	case "image/jpg":
 	case "image/jpeg":
-		if (res.size < (Helper.config.prefetchMaxImageSize * 1024)) {
+		if (res.text.length < (Helper.config.prefetchMaxImageSize * 1024)) {
```

Comment:
I think it's better to check actual size (since we have it now) rather than trusting the header, which may have invalid value.